housing_cooperative_base/models/lease.py

```diff
@@ -140,6 +150,28 @@ def _compute_state(self):
             else:
                 lease.state = "draft"
 
+    @api.multi
+    @api.depends("lease_line_ids")
```

Comment:
I think you need to depend on `lease_line_ids.premise_id`

Reply:
You found a bug! 👍 
This depends does currently not detect a change in the value of the 'is_arcade' field in a housing, not even if you extend it as you propose. I might rewrite this part (later) to put this field in the premise, which is the only way I currently see to point to the actual field in the depends... Ideas welcome.

Reply:
Just tried that and it didn't solve it. It's hard to make it check for a mapped field... Ideas?

Reply:
Let's have a chat about it.